phpstan-baseline.neon

```diff
@@ -4120,6 +4119,16 @@ parameters:
 			count: 1
 			path: LibreNMS/OS/Iosxe.php
 
+		-
+			message: "#^Property LibreNMS\\\\OS\\\\Iosxe\\:\\:\\$isis_codes has no type specified\\.$#"
+			count: 1
+			path: LibreNMS/OS/Iosxe.php
+
+		-
+			message: "#^Method LibreNMS\\\\OS\\\\Iosxe\\:\\:parseAdjacencyTime\\(\\) has parameter \\$data with no type specified\\.$#"
+			count: 1
+			path: LibreNMS/OS/Iosxe.php
+
```

Comment:
Could you please fix these issues instead?

Reply:
Should be fixed, can you double check I've done this correctly? PHPStan is not my forte... I mostly referenced other modules with exclusions.